src/compiler/utilities.ts

```diff
@@ -4658,6 +4658,10 @@ namespace ts {
         return ModifierFlags.None;
     }
 
+    export function createModifiers(modifierFlags: ModifierFlags): ModifiersArray | undefined {

```

Comment:
I don't see a consumer in the compiler layer.  Perhaps this should go in services/utilities?